ui/css/treeherder-job-buttons.css

```diff
@@ -51,19 +52,25 @@
 }
 
 .group-content {
-  margin-left: -3px;
+  margin: 0 0;
+  white-space: nowrap;
+  
```

Comment:
This empty line shouldn't be here; the closing `}` of the class properties should be moved up one line.